src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+import './SecondaryNav.css';
```

Comment:
🔥component styles get included automatically in `src/tailwind.src.css` via a `@import './components/**/*.css';`